cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -676,12 +677,14 @@ void InitEncoderPages(cudf::detail::device_2dspan<EncColumnChunk> chunks,
  * @param[out] comp_in Compressor input buffers
  * @param[out] comp_in Compressor output buffers
  * @param[out] comp_stats Compressor results
+ * @param[in] write_v2_headers True if V2 page headers should be written
```

Comment:
Should this "in" parameter be listed in the function signature before the "out" parameters? (Is this docstring correct about what it labels "in" and "out"?)

Reply:
I can move it up. I think all the docstrings in this file could use some love. I tried cleaning up some in a prior PR, but there's still more to do :(

Reply:
No worries, and thank you for leaving the code in a better state than you found it. 👍 I'll defer to @vuule on the decision of parameter order if he has other preferences. This is fine by me.

Reply:
AFAICT, the order is consistent with most functions in cuIO so I'm fine with the current location. Having `write_v2_headers` as the first parameter would somehow be more odd to me (even though in theory that's the right spot).

Reply:
should I revert the change to the param order then?

Reply:
it's fine :D